### PR TITLE
[BLOG] Add release-spyder-4beta2

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-release-spyder-4beta2-quansight.vercel.app/blog/release-spyder-4beta2).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/release-spyder-4beta2.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+light theme under  ``Preferences > Appearance`` by either choosing a light-background
+syntax-highlighting scheme, or changing ``Interface theme`` to ``Light``.
+
+![Screenshot of the Spyder main window with default panes, with the dark theme applied across the entire interface.](/posts/release-spyder-4beta2/spyder-qdarkstyle.png)
+
+Pretty, right :-) ?
+
+This enhancement was made possible by the work of
+[Colin Duquesnoy](https://github.com/ColinDuquesnoy),
+[Daniel Pizzeta](https://github.com/DPizzeta)
```
Comment (noatamir) [inline apps/labs/posts/release-spyder-4beta2.md:56]: This link leads to a 404. Should we remove it? Do we know if they have a new account we can link to?

Comment (rgommers) [reply apps/labs/posts/release-spyder-4beta2.md:56]: https://github.com/dpizetta

File: apps/labs/posts/release-spyder-4beta2.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+the theme dynamically.
+
+
+## Language Server Protocol architecture
+
+The Language Server Protocol (LSP) was created by Microsoft for Visual Studio Code to standardize how
+development tools (e.g. editors and IDEs) communicate with
+servers that provide code completion, linting and related facilities for
+different programming languages. With LSP, as
+[they describe it](https://microsoft.github.io/language-server-protocol/):
```
Comment (noatamir) [inline apps/labs/posts/release-spyder-4beta2.md:71]: The linked website doesn't have the original quote anymore. @goanpeca would you like to replace it with [an internet archive link](https://web.archive.org/web/20190528210156/https://microsoft.github.io/language-server-protocol/) that has the original quote, or keep it as is?
CC @trallard, do you have a preference for how to handle out-of-date links?

File: apps/labs/posts/release-spyder-4beta2.md (hunk 1 above)
Comment (noatamir) [inline apps/labs/posts/release-spyder-4beta2.md:56]:
```suggestion
[Daniel Pizzeta](https://github.com/dpizetta)
```

File: apps/labs/posts/release-spyder-4beta2.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
```
Comment (noatamir) [inline apps/labs/posts/release-spyder-4beta2.md:71]:
```suggestion
[they describe it](https://web.archive.org/web/20190528210156/https://microsoft.github.io/language-server-protocol/):
```

**Review by noatamir — APPROVED** [review]
LGTM. merging

**Pull request merged**